quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testStartDevHelp.unix.approved.txt

```diff
@@ -150,14 +150,14 @@ HTTP(S):
 --https-protocols <protocols>
                      The list of protocols to explicitly enable. Default: TLSv1.3,TLSv1.2.
 --https-trust-store-file <file>
-                     DEPRECATED: The trust store which holds the certificate information of the
-                       certificates to trust.
+                     DEPRECATED. Use: truststore-paths. -- The trust store which holds the
+                       certificate information of the certificates to trust.
 --https-trust-store-password <password>
-                     DEPRECATED: The password of the trust store file.
+                     DEPRECATED. Use: truststore-paths. -- The password of the trust store file.
 --https-trust-store-type <type>
-                     DEPRECATED: The type of the trust store file. If not given, the type is
-                       automatically detected based on the file name. If 'fips-mode' is set to
-                       'strict' and no value is set, it defaults to 'BCFKS'.
+                     DEPRECATED. Use: truststore-paths. -- The type of the trust store file. If not
```

Comment:
There is currently no replacement for type - if it's not a pkcs12 nor pem, it's not supported.

Reply:
I'll add a more descriptive note.

Reply:
Change the wording a bit and removed the hinted option.